src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+ul,
+a,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav_list {
+  display: flex;
+}
+
+.nav_item {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  font-family: Roboto, sans-serif;
```

Comment:
Roboto font doesn't exist by default
You should add it

That's a reason tests fail